src/views/open-api-docs-view/components/operation-header/operation-header.module.css

```
@@ -0,0 +1,22 @@
+.wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: 8px;
+}
+
+.heading {
+	composes: g-offset-scroll-margin-top from global;
+	composes: hds-typography-display-400 from global;
+	font-weight: var(--token-typography-font-weight-bold);
```

Quick fix for overflow issues on mobile:

```suggestion
	font-weight: var(--token-typography-font-weight-bold);
	overflow-wrap: break-word;
```